jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.vue

```diff
@@ -53,6 +53,15 @@
       hint="Select spectral region that defines the continuum."
     />
 
+    <plugin-dataset-select
+      v-if="continuum_subset_selected !== 'None'"
+      :items="continuum_dataset_items"
+      :selected.sync="continuum_dataset_selected"
+      :show_if_single_entry="false"
+      label="Continuum Spectrum"
+      hint="Select the spectrum used to visualize the continuum inputs.  The continuum will be recomputed on the input cube when computing the moment map."
+    />
```

Comment:
do we want to expose this to the user (in the UI and API)?  Or would it be better to hide this under-the-hood and just always use the first valid dataset found?

Reply:
If you are asking me, I have no opinion. I don't even know what this continuum business is about. Sounds like something unrelated to spectral extraction since it touches moment map plugin.

Reply:
As described in the description, the moment map plugin had to be changed because it used to use the spectrum representation of the input cube to visualize the continuum inputs (the continuum is then recomputed based on the actual cube input when computing the moment map).  With the new auto-extraction logic, we instead want to visualize the continuum on one of the 1d spectra... but which one?  Do we just assume the first entry that was extracted from the selected cube or do we allow the user to choose?

Reply:
Sounds like a question for POs?

Reply:
Yep - or any one that has thoughts/opinions!